htdocs/okapi_settings.php

```diff
@@ -13,7 +13,6 @@ function get_okapi_settings()
     # [2] http://code.google.com/p/opencaching-api/source/browse/trunk/okapi/settings.php
     # [3] http://code.google.com/p/opencaching-api/issues/detail?id=132
     date_default_timezone_set('Europe/Berlin');
-    $opt['rootpath'] = isset($GLOBALS['rootpath']) ? $GLOBALS['rootpath'] : __DIR__ . '/';
```

Comment:
Did you test this? Should produce errors, because `$opt['rootpath']` is used in the OC settings scripts, which are included in the next lines:

Reply:
Notice: Undefined index: rootpath in /mnt/ocde/htdocs/config2/settings-dist.inc.php on line 527

Reply:
fixed

Reply:
Looks like you just reverted your last change. The original line is back now, including the $GLOBALS['rootpath'] - which points to htdocs/vendor/opencaching/ instead of htdocs/.

Reply:
https://github.com/OpencachingDeutschland/oc-server3/pull/623/commits/8622f16fab2ec50de6f57884a0d414b8534ff466

Reply:
[okapi/index.php](https://github.com/teiling88/oc-server3/blob/feature/load-okapi-via-composer/htdocs/okapi/index.php) includes vendor/opencaching/okapi/index.php, and that overwrites `$GLOBALS['rootpath']` with `__DIR__.'/../'` == htdocs/vendor/opencaching/.

Therefore I suggested [this change](https://github.com/teiling88/okapi/pull/3/commits/65f13836b0baa5f5164490cf27a1c7c7b90d7acc) ...

Reply:
ah ok - can you please reopen the pr with this change?